android/src/main/java/com/magicleap/magicscript/utils/Vector2.kt

```diff
@@ -32,36 +32,6 @@ data class Vector2(var x: Float = 0F, var y: Float = 0F) {
         return Vector2(x - other.x, y - other.y)
     }
 
-    operator fun minus(other: Float): Vector2 {
```

Comment:
Are those operations unused (minus, times, div etc)?

Reply:
yes, in other case tests would fail :-)